### PR TITLE
Add Upollo Segment source documentation

### PR DESCRIPTION
**Pull request by stephen-up**

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/upollo/index.md [hunk 1]
```diff
@@ -1,4 +1,70 @@
...
----
\ No newline at end of file
+title: Upollo Source
+---
+
+[Upollo](https://upollo.ai?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} gives unique and actionable insights that lead to conversion, retention and expansion.
+
+This source is maintained by Upollo. For any issues with the source, [contact their Support team](mailto:support@upollo.ai).
+
+_**NOTE:** The Upollo Source is currently in beta, which means that they are still actively developing the source. If you are interested in joining their beta program or have any feedback to help improve the Upollo Source and its documentation, [let the Upollo team know](mailto:support@upollo.ai)._
```
Comment (cayden-uw) [inline src/connections/sources/catalog/cloud-apps/upollo/index.md:9]: Let's set a reminder to remove this so we don't forget

File: src/connections/sources/catalog/cloud-apps/upollo/index.md [hunk 2]
```diff
@@ -1,4 +1,70 @@
...
+5. Copy the Write key from the Segment UI.
+6. Log in to your Upollo account - navigate to [Connections](https://upollo.ai/app/settings/connections) > Connect Segment Source and paste the key to connect.
+
+
+## Stream
+
+Upollo uses server-side `identify` calls to send information about users. The table below describes the fields.
+Upollo uses our stream Source component to send Segment event data. It uses a server-side `identify` method to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL.
+
+## Event Properties
```
Comment (cayden-uw) [inline src/connections/sources/catalog/cloud-apps/upollo/index.md:29]: Don't we also send flag information?

Comment (stephen-up) [reply src/connections/sources/catalog/cloud-apps/upollo/index.md:29]: Ah, thanks yes we do.

Comment (stephen-up) [reply src/connections/sources/catalog/cloud-apps/upollo/index.md:29]: Added.

File: src/connections/sources/catalog/cloud-apps/upollo/index.md (hunk 1 above)
Comment (stephen-up) [reply src/connections/sources/catalog/cloud-apps/upollo/index.md:9]: Yea. I've created an issue for it.
UP-3182

**Review by cayden-uw — APPROVED** [review]

**Comment by stephen-up** [comment]
Closing this PR to submit directly upstream.

**Pull request closed without merging**